lib/GallerySwiper.js

```diff
@@ -89,7 +89,7 @@ var GallerySwiper = function (_Component) {
                 newLazyload = nextProps.lazyLoad;
 
 
-            if (images !== newImages) {
+            if (images !== newImages || images.length !== newImages.length) {
```

Comment:
Yeah, every time I write this comment I look up and see it's generated code... I keep going down to find the actual code :)